modules/log6x-loki-sizing.adoc

```diff
@@ -0,0 +1,88 @@
+// Module is included in the following assemblies:
+// * observability/logging/logging-6.0/
+
+:_mod-docs-content-type: CONCEPT
+[id="log6x-loki-sizing_{context}"]
+= Loki deployment sizing
+
+Sizing for Loki follows the format of `1x.<size>` where the value `1x` is number of instances and `<size>` specifies performance capabilities.
+
+[IMPORTANT]
+====
+It is not possible to change the number `1x` for the deployment size.
+====
+
+.Loki sizing
+[cols="1h,5*",options="header"]
+|===
+|
+|1x.demo
+|1x.pico [6.1+ only]
+|1x.extra-small
+|1x.small
+|1x.medium
+
+|Data transfer
+|Demo use only
+|50GB/day
+|100GB/day
+|500GB/day
+|2TB/day
+
+|Queries per second (QPS)
+|Demo use only
+|1-25 QPS at 200ms
+|1-25 QPS at 200ms
+|25-50 QPS at 200ms
+|25-75 QPS at 200ms
+
+|Replication factor
+|None
+|2
+|2
+|2
+|2
+
+|Total CPU requests
+|None
+|6.8 vCPUs
```

Comment:
how about use 7 here?

Reply:
That is a approximate value. I think it is better to be a little bigger than the actual value.